lib/util/devutil.js

```diff
@@ -191,4 +181,18 @@ devutil.makeIdentity = function(serial, properties) {
         , ram: ram
     }
 }
+
+devutil.getDeviceMarketName = function(adb, serial) {
+    return adb.getDevice(serial).execOut('settings get global device_name', 'utf-8').then(function(deviceName) {
```

Comment:
in some cases this field contains values like 
<img width="374" alt="image" src="https://github.com/user-attachments/assets/0f5b56f6-8467-434d-8b1f-68e92daca80d" />